Branch-SDK/Branch-SDK/Core/Branch.m

```diff
@@ -354,6 +357,16 @@ - (BOOL)handleDeepLink:(NSURL *)url {
     return handled;
 }
 
+- (BOOL)continueUserActivity:(NSUserActivity *)userActivity {
+    NSString *linkIdentifier = [self.contentDiscoveryManager spotlightLinkIdentifierFromActivity:userActivity];
```

Comment:
don't we need to set this in the preference helper so that it's sent back on /v1/install or /v1/open?